README.md

```diff
@@ -15,7 +15,7 @@
 - qt5-qtbase-devel (install by `yum`)
 - xorg-x11-server-Xvfb (optional, install by `yum`)
 - qasync (install by `conda -c conda-forge`)
-- [black](https://github.com/psf/black) (22.12.0, optional)
+- [black](https://github.com/psf/black) (23.1.0, optional)
 - [flake8](https://github.com/PyCQA/flake8) (4.0.1, optional)
 - [isort](https://github.com/PyCQA/isort) (5.10.1, optional)
```

Comment:
Please update  to `5.12.0`.

Reply:
Done in the commit: [ad8e2e3](https://github.com/lsst-ts/ts_m2gui/pull/34/commits/ad8e2e32b04894e7bf312394b459378a4110f482). Thanks!